Maestro/src/Microsoft.DotNet.Maestro.WebApi/Models/ModifiedFileModel.cs

```diff
@@ -16,11 +17,14 @@ public class ModifiedFileModel
 
         public string CommitId { get; set; }
 
+        public string CommitList { get; set; }
+
         public static bool TryParse(
             string repoFullName,
             string refSpec,
             string fullPath,
             Commit commit,
+            List<Commit> Commits,
```

Comment:
In addition to @eerhardt's changes I'd also suggest changing this to `IEnumerable<Commit>`.